falco/client.py

```diff
@@ -1,23 +1,46 @@
 import grpc
 
 from falco.client_credentials import get_grpc_channel_credentials
-from falco.domain import Response
-from falco.svc.output_pb2_grpc import serviceStub
+from falco.domain import Response, Request
+from falco.svc.outputs_pb2_grpc import serviceStub
+from falco.svc.version_pb2_grpc import serviceStub as versionServiceStub
+from falco.schema.version_pb2 import request as versionRequest
```

Comment:
In my opinion the Client should only use domain models

Reply:
I took a stab at the refactor you're suggesting and got pretty confused. Would you be willing to throw a patch together to move the code into the domain model?